### PR TITLE
[Cypress] - fix padded coded regions tests and add regionId support

### PR DESCRIPTION
**Pull request by SophieDePaula**

### CODE REVIEW

File: js/packages/driver/src/element.ts [hunk 1]
```diff
@@ -55,7 +55,9 @@ export class Element<TDriver, TContext, TElement, TSelector> {
       throw new TypeError('Element constructor called with argument of unknown type!')
     }
 
-    if (this._selector && this._spec.untransformSelector) {
+    if (specUtils.isSimpleCommonSelector(this._selector)) {
+      this._commonSelector = this._selector.selector
```
Comment (SophieDePaula) [inline js/packages/driver/src/element.ts:59]: @kyryloonufriiev I have a problem here with the type, can you take a look, I am not sure how to fix this.

Comment (kyryloonufriiev) [reply js/packages/driver/src/element.ts:59]: You should not do `this._selector.selector`, just `this._selector.selector`, I understand that you are addressing the fact that for some reason all of the selectors in cypress look like `{selector
: {selector: 'blabla'}}`. But you should not have it in the first place.

Comment (SophieDePaula) [reply js/packages/driver/src/element.ts:59]: @kyryloonufriiev did you mean that is should just be `this._commonSelector = this._selector`?
That's fine, but still, there is a problem with the `type`

**Comment by SophieDePaula** [comment]
@kyryloonufriiev this is ready for review.

File: pending-changes.yaml [hunk 2]
```diff
@@ -77,6 +77,7 @@ Apollo SDK:
 "@applitools/eyes-cypress":
   feature:
     - Add support for padded coded regions
+    - Add support for regionId
```
Comment (SophieDePaula) [inline pending-changes.yaml:80]: @benbabayoff8

Comment (benbabayoff8) [reply pending-changes.yaml:80]: In other SDKs we wrote:"Add support for dynamic coded regions", so let's be consistent :)

File: js/packages/eyes-cypress/src/browser/socketCommands.js [hunk 3]
```diff
@@ -54,6 +54,11 @@ function socketCommands(socket, refer) {
     return await spec.getCookies();
   });
 
+  socket.command('Driver.untransformSelector', async selector => {
+    const result = await spec.untransformSelector(selector);
+    return result;
+  });
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/browser/socketCommands.js:60]: Do we need it?

**Review by kyryloonufriiev — APPROVED** [review]
Looks good to me. I  just think you forget to remove something. please see my comment.

File: pending-changes.yaml (hunk 2 above)
Comment (SophieDePaula) [reply pending-changes.yaml:80]: Yes sir!

File: js/packages/eyes-cypress/src/browser/socketCommands.js (hunk 3 above)
Comment (SophieDePaula) [reply js/packages/eyes-cypress/src/browser/socketCommands.js:60]: No, I forgot to remove it. Thanks

**Pull request merged**